runtime/codegen/metrics.go

```diff
@@ -42,18 +42,33 @@ var (
 	methodLatencies = metrics.NewHistogramMap[MethodLabels](
 		MethodLatenciesName,
 		"Duration, in microseconds, of Service Weaver component method execution",
-		metrics.NonNegativeBuckets,
+		nonNegativeBuckets,
 	)
 	methodBytesRequest = metrics.NewHistogramMap[MethodLabels](
 		MethodBytesRequestName,
 		"Number of bytes in Service Weaver component method requests",
-		metrics.NonNegativeBuckets,
+		nonNegativeBuckets,
 	)
 	methodBytesReply = metrics.NewHistogramMap[MethodLabels](
 		MethodBytesReplyName,
 		"Number of bytes in Service Weaver component method replies",
-		metrics.NonNegativeBuckets,
+		nonNegativeBuckets,
 	)
+
+	// nonNegativeBuckets provides rounded bucket boundaries for the autogenerated histograms.
+	nonNegativeBuckets = []float64{
```

Comment:
Can this be defined in `weaver/internal/metrics.go`?

If so, maybe best to put it that so that it can be shared. If not, it's fine as you have it.

Reply:
Good point. It requires a little bit of massaging though.